### ISSUE TITLE
Add honors icon to sections in SchedulePreview

### ISSUE THREAD

**Issue by gannonprudhomme** [aggie-coding-club/Rev-Registration#237]
At the moment, if a section in a schedule is online, there's no indication that it's online, and as such it is kind of confusing when it says you're in a section for it and it doesn't show up on the actual Schedule / show a MeetingCard for it. As such, we should display an icon next to the section in the SchedulePreview that indicates that it's an online course (same thing with honors).

**Comment by gannonprudhomme** [comment]
Do we have a current design for this? If not we should make one before we start working on this

**Comment by ryan-conn** [comment]
Here's a mockup of the buttons (apologies for them looking pretty bad, I did the mockup in MS paint):
![image](https://user-images.githubusercontent.com/28655462/94502722-671e6980-01ca-11eb-9260-5fca97655ec7.png)
- See the [Material UI Icons page](https://material-ui.com/components/material-icons/) for how to actually use import them in your code, in particular they are `<LanguageIcon>` for online sections and `<HonorsIcon>` for honors sections.
- You'll need to change `autoscheduler/frontend/src/components/SchedulingPage/SchedulePreview/ScheduleListItem/ScheduleListItem.tsx` to render the appropriate icons for if a section is online or honors. Here's the relevant part of the code that creates the boxes showing the colors of meetings in the schedule preview, subject, course number, and section number:
![image](https://user-images.githubusercontent.com/28655462/94503410-0728c280-01cc-11eb-92cd-5949079fa8eb.png)
- After the text, you'll have to check if a section is honors or online, and put the appropriate icons. How can you find that out? If you look at `sec` in the above code, you'll see that it's a `Section`. Going to `autoscheduler/frontend/src/types/Section.ts`, you can see the `honors` and `web` properties that you can check:
![image](https://user-images.githubusercontent.com/28655462/94503623-7d2d2980-01cc-11eb-882c-030cac1d9c5c.png)
- Add [Material UI Tooltips](https://material-ui.com/components/tooltips/) to the icons so that users can hover over them and see what they mean. The honors tooltip should say "Honors", and the online tooltip should say "Online - Asynchronous'.

**Comment by gannonprudhomme** [comment]
By the way @fool1280 , we typically close the issues once the pull requests are merged into `master`. Also, if you add "Closes #237" in your PR for this, once the PR is merged in master the issue will be closed automatically.

**Comment by gannonprudhomme** [comment]
I'm not sure if we originally planned on this, but I think this should also include moving sections with no meeting times to the bottom of the list of sections in the SchedulePreview and remove their color icon square thing, to indicate that they're not going to be displayed on the actual schedule.
